rpcs3/Emu/RSX/rsx_methods.cpp

```diff
@@ -1259,6 +1259,11 @@ namespace rsx
 			LOG_TRACE(RSX, "NV0039_OFFSET_IN: pitch(in=0x%x, out=0x%x), line(len=0x%x, cnt=0x%x), fmt(in=0x%x, out=0x%x), notify=0x%x",
 				in_pitch, out_pitch, line_length, line_count, in_format, out_format, notify);
 
+			if (!line_count || !line_length)
+			{
+				return;
```

Comment:
Should emit something in log for this condition. A warning would be best I think.